osquery/tables/system/darwin/authorizations.mm

```diff
@@ -131,10 +131,13 @@ QueryData genAuthorizations(QueryContext &context) {
       }
 
       CFIndex count = CFDictionaryGetCount(rightSet);
-      const void *keys[count];
-      const void *values[count];
-      CFDictionaryGetKeysAndValues(rightSet, keys, values);
-      if ((const void *)keys == nullptr || (const void *)values == nullptr) {
+      std::vector<const void *> keys;
+      keys.resize(count);
+      std::vector<const void *> values;
+      values.resize(count);
+      CFDictionaryGetKeysAndValues(rightSet, keys.data(), values.data());
+      if ((const void *)keys.data() == nullptr ||
```

Comment:
Why do you have to explicit cast these? Shouldn't `.data()` be a `const void*` type? http://en.cppreference.com/w/cpp/container/vector/data